src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>3 Colored blocks</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body class="forBody">
```

Comment:
use only lowercase letters for css classes, use `-` or `_` to separate words

forBody - not a good name, will be better to name `container` etc.